src/main/java/com/backbase/ct/bbfuel/client/user/UserIntegrationRestClient.java

```diff
@@ -23,11 +23,13 @@
 
     private final BbFuelConfiguration config;
     private static final String SERVICE_VERSION = "v2";
+    private static final String SERVICE = "client-api";
```

Comment:
Why do we use "client-api" in User**Integration**RestClient?

Reply:
Without the addition of "client-api" the services are returning 404.

Reply:
We have `UserPresentationRestClient.java` for calling `client-api` and `UserIntegrationRestClient` for calling `integration-api`.
I do not understand why do we mix responsibilities.

Reply:
Indeed we do, but it doesn't work. Please try on this endpoint if you want to be sure:
http://user.2383.dbs.backbase.test/integration-api/v2/users

Reply:
So our integration API is exposed via `/client-api` as well (I just thought we are on the lean architecture already). 
Please rename the constant to be `CLIENT_API`(like others).